packages/javascript/bh-shared-ui/src/components/HelpTexts/ADCSESC4/ADCSESC4.tsx

```diff
@@ -19,13 +19,15 @@ import WindowsAbuse from './WindowsAbuse';
 import LinuxAbuse from './LinuxAbuse';
 import Opsec from './Opsec';
 import References from './References';
+import Composition from '../ADCSESC6a/Composition';
```

Comment:
This is neat. It seems like a good idea to make this component generally available instead of one for each edge. This seems good for now to avoid duplication as there is a lot to be desired in refactoring the edge info content which is out of scope for this changeset. I appreciate avoiding creating more cruft 🙌